pkg/mkconf/Dockerfile

```diff
@@ -11,3 +11,5 @@ COPY --from=build /out /
 
 WORKDIR /
 ENTRYPOINT [ "/make-config" ]
+
```

Comment:
This is dummy update. But it helps us to check what will happen in case of real update. And we see the problem. In packages workflow we build packages required for tools on arm64 for host arch (again, arm64). But in composing job we need packages for another host arch (amd64) and they are not available.
I think we should build packages required for tools in runner (job) where we do composing.

Reply:
Exactly. Once it all runs clean, I will remove that commit and push again.